src/services/at-sce-api.service.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * AT SCE UI - AT SCE Service API.
+ * Copyright 2021 AgileThought, Inc.
+ *
+ * Functions for at-sce-api.service endpoint.
+ *
+ * @author @at-internship
+ * @version 1.0
+ *
+ */
+// Constants
+const axios = require("axios");
+const AT_SCE_SERVICE = {};
+
+// LOCAL
+require("dotenv").config();
+const AT_SCE_SERVICE_URI = process.env.AT_SCE_SERVICE_URI || `https://at-sce-api.herokuapp.com/api`;
```

Comment:
this can be avoided by setting an env file in your local environment, prod url shouldn't be there and the env variable should be consumed instead.